### PR TITLE
[Basic UI] Basic random board served at GET /board

### PR DESCRIPTION
**Pull request by bcollazo**
UI to visualize a random board.

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect } from "react";
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
+  useEffect(() => {
+    (async () => {
+      const response = await fetch("http://localhost:5000/board");
```
Comment (tonypr) [inline ui/src/App.js:8]: consider extracting this url as a constant.
i.e.
`host = http://localhost:5000`.

Comment (pachewise) [reply ui/src/App.js:8]: se pone en el index.jsx o en el config... pero por ahora whatever

File: ui/src/App.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+      const data = await response.json();
+      console.log(data);
```
Comment (tonypr) [inline ui/src/App.js:10]: remove console log?

File: ui/src/App.js (hunk 1 above)
Comment (pachewise) [reply ui/src/App.js:8]: @bcollazo esto debería ser App.jsx para representar que usa React JSX

**Comment by bcollazo** [comment]
Noted! Thanks for comments. Addressed and finalized for merge.

https://www.loom.com/share/7737cfe9d3384c4f8bf164dffc79a0c7

**Pull request merged**